v20.2/backup.md

```diff
@@ -111,13 +111,19 @@ For an example of an incremental backup, see the [Create incremental backups](#c
 
 The `BACKUP` process minimizes its impact to the cluster's performance by distributing work to all nodes. Each node backs up only a specific subset of the data it stores (those for which it serves writes; more details about this architectural concept forthcoming), with no two nodes backing up the same data.
 
-For best performance, we also recommend always starting backups with a specific [timestamp](timestamp.html) at least 10 seconds in the past. For example:
+`BACKUP`, like any read, cannot export a range if it contains an unresolved intent. If CockroachDB hits an intent during iteration, CockroachDB will typically return a `WriteIntentError` and let the store resolve it and retry the request before replying to the sender.
+
+However, blocking intent resolution can mean a single transaction left open can cause backups to hang, sometimes for hours or even days in some observed cases. While you typically want bulk, background jobs like `BACKUP` to have as little impact on your foreground traffic as possible, it's more important for backups to actually complete (which maintains your [recovery point objective (RPO)](https://en.wikipedia.org/wiki/Disaster_recovery#Recovery_Point_Objective)). So while you typically would not want a `BACKUP` to immediately run at high priority and abort any transactions from OLTP foreground traffic as soon as it starts, you do, eventually, want it to do whatever it needs to do to finish.
```

Comment:
This paragraph taken from the commit message is mostly talking about the _pre_ 20.2 (mis)behavior as motivation for that commit -- "a single transaction left open can cause backups to hang, sometimes for hours or even days" is specifically _not_ a problem in 20.2 thanks to that change. I almost feel like we could elide the paragraph entirely and/or work the reason for invoking priority (RPO) into another sentence elsewhere.